doc/contributing/primordials.md

```diff
@@ -10,6 +10,7 @@ important than reliability against prototype pollution:
 * `node:http`
 * `node:http2`
 * `node:tls`
+* `node:zlib`
```

Comment:
what's the rationale here for why perf and readability matters more for zlib than robustness?

Reply:
The question I ask myself, why does zlib matters more for robustness compared to `node:http`?

Reply:
I don't think unreadable code in the name of robustness is not worth it in the context of `node:zlib`.

Reply:
Why?